projects/noloan/app/src/main/java/hasadna/noloan/mainactivity/MainActivity.java

```diff
@@ -49,6 +59,22 @@ protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.main_activity);
 
+    // Firebase authentication
+    firebaseAuth = FirebaseAuth.getInstance();
```

Comment:
I see you have this code twice - here and in `DbMessages`. Try to see if you can avoid that.

Reply:
Thanks. It's there from when I first tried to login the user.... 
Removed..